app/grandchallenge/workstations/models.py

```diff
@@ -547,3 +557,35 @@ def save(self, *args, **kwargs) -> None:
                     queue=f"workstations-{self.region}",
                 )
             )
+
+
+def feedback_screenshot_filepath(instance, filename):
+    return (
+        f"session-feedback/"
+        f"{instance.session.pk}/"
+        f"{get_valid_filename(filename)}"
+    )
+
+
+class Feedback(UUIDModel):
+    session = models.ForeignKey(Session, on_delete=models.CASCADE)
+    screenshot = models.ImageField(
+        upload_to=feedback_screenshot_filepath,
+        storage=protected_s3_storage,
+        blank=True,
+    )
+    user_comment = models.TextField()
+    context = models.TextField(blank=True)
+
+    def save(self, *args, **kwargs) -> None:
+        created = self._state.adding
```

Comment:
Nit: the variable should be `adding`, not `created` (got it confused with an attribute on the UUID Model)